sdk/storage/examples/data_lake_00_file_system.rs

```diff
@@ -42,6 +42,8 @@ async fn main() -> Result<(), Box<dyn Error + Send + Sync>> {
         .clone()
         .into_file_system_client(file_system_name.to_string());
 
+    // =============================================================================================
```

Comment:
I'd like to avoid these.

Reply:
Yes, I'm not a fan either. Trying find the best way to indicate what the main part of the test is, vs setup/teardown. Reading up on how to structure tests in Rust, but haven't found anything good so far. Also, no other e2e tests seem to have this much coverage so the problem hasn't arisen there it seems.